api/apiconsumer/models.py

```diff
@@ -26,6 +26,7 @@ class APIConsumer(models.Model):
     description = models.TextField()
     token = models.CharField(max_length=200, unique=True, default=generate_key)
     permission_level = models.IntegerField(default=2)
+    is_active = models.BooleanField(default=False)
```

Comment:
Ensure that old tokens are marked with `is_active` to true before deploying, just in case we do checks with this later.

Reply:
To do this, you want to write a `RunPython` migration operation.

Reply:
will do!